modules/imgimport/include/imgimport.h

```diff
@@ -33,11 +33,13 @@
 #define IMAGE_IMPORT_H_INCLUDED
 
 
-#include "frame.h"
+#include "../../core/include/frame.h"
```

Comment:
This should be avoided since it will break if either file is moved, instead you should add the core/include directory to the include directories in the CMakeLists (and it is already there, though I notice that the CMakeLists is using the relative path, which should really be replaced with a constant, but that's another issue).